conduct/conduct_enforcement.md

```diff
@@ -0,0 +1,117 @@
+# Jupyter Code of Conduct - Enforcement Manual
+
+This is the enforcement manual followed by Jupyter's Code of Conduct
+Committee. It's used when we respond to an issue to make sure we're consistent
+and fair. It should be considered an internal document, but we're publishing it
+publicly in the interests of transparency.
+
+
+## The Code of Conduct Committee
+
+All responses to reports of conduct violations will be managed by a Code of
+Conduct Committee ("the committee").
+
+The Jupyter Steering Council will establish this committee, comprised of at
```

Comment:
It is probably worth adding that the membership and chair of this committee are public knowledge.